### DISCUSSION TITLE
axis添加title的DisplayObject类型的值显示错误

### DISCUSSION THREAD

**Discussion by Keithcaiqian** [antvis/G2#5874] [Q&A]
## 代码
![image](https://github.com/antvis/G2/assets/54406776/8c28579b-641e-46f7-a0af-ed099a3dabc9)
## 显示效果
![image](https://github.com/antvis/G2/assets/54406776/12b53d2e-0e8e-449d-b85a-cd9c48b477ea)
![image](https://github.com/antvis/G2/assets/54406776/61587107-d6c4-427a-8072-98b8433e9545)

### 平台

- 操作系统: [macOS, Windows, Linux, React Native ...]
- 网页浏览器: [Google Chrome, Safari, Firefox]

**Comment by pearmini** [comment]
目前 title 只支持 string，不支持 G 的 DisplayObject 哈

**Reply by Keithcaiqian** [reply]
那现在是不是没办法加一些自定义的样式。比如加一个小三角然后监听点击事件。
